lib/binding.js

```diff
@@ -3,18 +3,20 @@
  */
 
 var errors = require('./errors');
+var path   = require('path');
 
 /**
  * Require binding
  */
 module.exports = function(ext) {
-  if (!ext.hasBinary(ext.getBinaryPath())) {
+  var binaryPath = ext.getBinaryPath();
+  if (!ext.hasBinary(binaryPath)) {
     if (!ext.isSupportedEnvironment()) {
       throw new Error(errors.unsupportedEnvironment());
     } else {
       throw new Error(errors.missingBinary());
     }
   }
-
-  return require(ext.getBinaryPath());
+  if (!path.isAbsolute(binaryPath)) binaryPath = path.join('../../..', binaryPath);
```

Comment:
Please keep to the existing code style.

```js
if (!path.isAbsolute(binaryPath)) {
  binaryPath = path.join('../../..', binaryPath);
}
```

Reply:
We can't assume *nix

```js
binaryPath = path.join('..', '..', '..', binaryPath);
```

Reply:
What is the rationale for going up exactly 3 directories?

Reply:
I make the assumption that non-absolute binary paths are relative to the working project directory (and I specify this in the docs). Consider the following project structure:

```
project
|-index.html
|-index.js
|-my-node-binding
  |-binding.node
|-node_modules
  |-node-sass
    |-lib
      |-binding.js
|-package.json
|-package-lock.json
```

You can see binding.js is 3 directories deep relative to the working project directory (node-modules/node-sass/lib). The ``require()`` function loads modules/files relative to the calling file. Since binding.js ``require()``'s the binary, we have to go up 3 directories.

Reply:
This doesn't account for nested `node_modules` for instance.

```
project
|-index.html
|-index.js
|-my-node-binding
  |-binding.node
|-node_modules
  |-node-sass@4.5.2
    |-lib
      |-binding.js
  |-gulp-sass
    |-node_modules
      |-node-sass@3.7.1
        |-lib
          |-binding.js
|-package.json
|-package-lock.json
```

npm does not guarantee flat node_modules, and it's not possible to make such a guarantee when version conflicts happen. I think `process.cwd()` is probably a better fit.